src/components/parameter/ParameterEdit.vue

```diff
@@ -269,7 +270,16 @@
                     <v-divider />
                     <v-row class="mx-0 py-1">
                       <v-col class="text-center" cols="2">
-                        <v-icon large right v-text="'mdi-alert-outline'" />
+                        <v-icon
+                          :small="state.options.iconSize != null ? true : false"
+                          :large="state.options.iconSize != null ? false : true"
```

Comment:
My bad -as suggested, this should be a bit better:

```suggestion
                          :small="state.options.iconSize === 'small'"
                          :large="state.options.iconSize === 'large'"
```